sky/clouds/service_catalog/common.py

```diff
@@ -189,7 +189,7 @@ def _get_all_supported_regions_str() -> str:
 
     filter_df = df
     if region is not None:
-        filter_df = filter_df[filter_df['Region'].str.lower() == region.lower()]
+        filter_df = _filter_region_zone(filter_df, region, None)
```

Comment:
Style nit: IIRC, the convention is to use kwarg when the provided argument is literal, isn't it?
```suggestion
        filter_df = _filter_region_zone(filter_df, region, zone=None)
```
Same for L579 and L594.

Reply:
Good point! Fixed. Thanks!